src/test/java/integration/ShipmentControllerIT.java

```diff
@@ -251,6 +253,25 @@ public void updateShipment() throws Exception {
     }
 
     @Test
+    @SuppressWarnings("unchecked")
+    public void removeShipmentGroup() throws Exception {
+        ShipmentGroup shipmentGroup = testHelper.createShipmentGroup();
+        Shipment shipment = testHelper.createShipment(shipmentGroup);
+        // update
+        given().
+                contentType(APPLICATION_JSON_VALUE).
+                queryParam("token", shipment.getSender().getCounterparty().getUser().getToken()).
+        when().
+                delete("/shipments/{uuid}/shipment-group", shipment.getUuid().toString()).
+        then().
+                body("shipmentGroupUuid", equalTo(null)).
+                statusCode(SC_OK);
+
+        testHelper.deleteShipment(shipment);
+        testHelper.deleteShipmentGroup(shipmentGroup);
```

Comment:
Plz, move it to testHelper.deleteShipment(shipment)
testHelper.deleteShipment() should do something like this
// delete shipment here ...
if (shipment.getShipmentGroup() != null) {
    deleteShipmentGroup(shipment.getShipmentGroup())
}